src/components/Reports/ReportEdit.tsx

```diff
@@ -40,8 +45,30 @@ export default function ReportEdit(): JSX.Element {
     } else {
       snackbar.toastError(strings.GENERIC_ERROR, strings.REPORT_COULD_NOT_OPEN);
     }
+
+    if (intervalref.current !== null) {
+      return;
+    }
+    intervalref.current = window.setInterval(getReport, 600);
+
+    return () => {
+      if (intervalref.current) {
+        window.clearInterval(intervalref.current);
+        intervalref.current = null;
+      }
```

Comment:
we could extract this out since it is used in multiple places..
btw, saw this earlier https://usehooks-ts.com/react-hook/use-interval in case it helps.. also the NotificationsDropdown clears intervals as well if you want to see how we do it there.
in case it helps with consistency..